webservices/resources/candidate_aggregates.py

```diff
@@ -165,11 +166,14 @@ def filter_range_fields(self, model):
         ('has_raised_funds', models.CandidateTotal.has_raised_funds),
         ('federal_funds_flag', models.CandidateTotal.federal_funds_flag),
         ('election_full', models.CandidateTotal.is_election),
+        # ('candidate_inactive', models.CandidateHistoryWithFuture.candidate_inactive),
     ]
 
+    #@print_query
     def build_query(self, **kwargs):
         history = models.CandidateHistoryWithFuture
         query = db.session.query(
+            #history
```

Comment:
Why is this commented out? Should it be removed?

Reply:
Sorry if this is a comment - I thought it was commented out code. If it's a comment, I don't think it adds value here.

Reply:
cleaned. thx.